common/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "private": true,
+  "name": "univaf-common",
+  "version": "0.1.0",
+  "description": "Common tooling for other parts of UNIVAF.",
+  "exports": {
+    ".": "./src/index.js",
+    "./*": "./src/*.js"
+  },
+  "types": "./dist/src/index.d.js",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/src/index.d.js"
+      ],
+      "./*": [
+        "./dist/src/*"
+      ]
+    }
+  },
+  "peerDependencies": {
+    "jest": ">=28.0.0"
+  },
+  "devDependencies": {
+    "typescript": "4.8.2"
+  },
+  "scripts": {
+    "clean": "tsc --build --clean",
```

Comment:
`--clean` is a new thing when you use `--build`. Should see about using it in the `clean` script for the server package.